packages/ndla-tabs/src/FilterTabs.tsx

```diff
@@ -56,16 +92,20 @@ class FilterTabs extends Component {
   }
 
   updateTabSizes() {
-    if (!this.tabWidths && this.liRefs[Object.keys(this.liRefs)[0]].parentNode.offsetWidth !== 0) {
+    const firstLi = this.liRefs[Object.keys(this.liRefs)[0]];
+    //@ts-ignore No way to know that parentNode has offsetWidth
+    if (!this.tabWidths && firstLi?.parentNode?.offsetWidth !== 0) {
       // Get all tabs widths
       this.tabWidths = [];
       let widestNode = 0;
       this.props.options.forEach((option, counter) => {
-        const nodeWidth = this.liRefs[option.value].parentNode.offsetWidth;
+        //@ts-ignore
+        const nodeWidth = this.liRefs[option.value]?.parentNode?.offsetWidth;
         widestNode = Math.max(nodeWidth, widestNode);
-        this.tabWidths[counter] = nodeWidth;
+        this.tabWidths![counter] = nodeWidth;
```

Comment:
if-statement i stedet for non-null assertion?

Reply:
La inn en if-check. Variabelen blir jo instansiert til å være et tomt array et par linjer lenger opp i funksjonen, så tenkte dette var den enkleste løsningen. Men den kan jo bli nullet i `componentDidUpdate` når som helst.